Source/WebCore/WebCore.xcodeproj/project.pbxproj

```diff
@@ -3,7 +3,7 @@
 	archiveVersion = 1;
 	classes = {
 	};
-	objectVersion = 54;
+	objectVersion = 51;
```

Comment:
This is unrelated.  Let's not change this.

Reply:
Fixed